packages/eslint-plugin-office-addins/src/rules/load-sync-read.ts

```diff
@@ -0,0 +1,81 @@
+import { TSESTree } from "@typescript-eslint/experimental-utils";
+import { Variable } from "@typescript-eslint/experimental-utils/dist/ts-eslint-scope";
+import { findReferences, OfficeApiReference } from "../utils";
+
+export = {
+  name: "load-sync-read",
+  meta: {
+    type: <"problem" | "suggestion" | "layout">"suggestion",
+    messages: {
+      loadSyncRead:
+        "Call load on '{{name}}' for '{{loadValue}}' followed by context.sync() before reading the object or its properties",
+    },
+    docs: {
+      description:
+        "Always call load on an object followed by a sync before reading it or one of its properties.",
```

Comment:
I think we need to be clearer about calling the sync after the load call and not just the object with the property.

Reply:
Added the context.sync() function to the message. Tell me if there are any suggestions

Reply:
I think the descript is still unclear about what it is checking.